src/fcntl.rs

```diff
@@ -584,17 +679,27 @@ pub fn fcntl(fd: RawFd, arg: FcntlArg) -> Result<c_int> {
     Errno::result(res)
 }
 
+/// Operations for use with [`flock`].
```

Comment:
The `flock()` function is deprecated so let's use the new `Flock::lock()` interface:
```suggestion
/// Operations for use with [`Flock::lock()`].
```